roles/openshift_openstack/tasks/unprovision.yml

```diff
@@ -1,4 +1,25 @@
 ---
+- name: Get kuryr services
+  delegate_to: "{{ groups.oo_first_master.0 }}"
+  oc_obj:
+    kind: endpoints
+    state: list
+    all_namespaces: true
+  register: svc_output
+
+- name: Delete kuryr LoadBalancers (cascading)
+  os_lbaas_deletion:
+    lbaas_annotation: "{{ item.metadata.annotations }}"
+  with_items: "{{ svc_output.results.results[0]['items'] }}"
+  when:
```

Comment:
We should make it conditional on using Kuryr. As it is, this will fail for deployments with OpenshiftSDN. They have no annotations and there's no loadbalancer associated with them.

Adding these two checks worked for me:

    - openshift_use_kuryr|default(false) == true
    - item.metadata.annotations is defined